src/Orleans/Core/ClientBuilder.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.CodeGeneration;
+using Orleans.Messaging;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Serialization;
+using Orleans.Streams;
+
+namespace Orleans
```

Comment:
Sure, maybe some other, although Hosting doesn't make much sense for the client library either.
Don't necessarily feel strong about Configuration, but what's the reason you don't like it? Effectively it's how you configure the client, right? (and eventually we'll fully configure it using the builder, other than the current `ClientConfiguration` object